engine/runtime/src/main/java/org/enso/interpreter/runtime/scope/TopLevelScope.java

```diff
@@ -145,15 +118,12 @@ private static Module getModule(
         throws ArityException, UnsupportedTypeException, UnknownIdentifierException {
       String moduleName = Types.extractArguments(arguments, String.class);
 
-      if (moduleName.equals(Builtins.MODULE_NAME)) {
-        return scope.builtins.getModule();
-      }
-      Module module = scope.modules.get(moduleName);
-      if (module == null) {
+      var module = scope.getModule(moduleName);
+      if (module.isEmpty()) {
         throw UnknownIdentifierException.create(moduleName);
       }
 
-      return module;
+      return module.get();
```

Comment:
I'd throw a custom error here.

Reply:
There is a check if the option is filled above.